avocado/plugins/multiplex.py

```diff
@@ -35,23 +34,28 @@ def __init__(self, *args, **kwargs):
 
     def configure(self, parser):
         parser = super(Multiplex, self).configure(parser)
+        parser.add_argument("--summary", type=int, help="Verbosity of "
```

Comment:
Well I thought about enum (short, medium, long, extensive) but with multiple plugins we don't know what's the limit and what's the usage (eg. for yaml it's actually a bitmask of different informations to show). How about making the argument optional, by default use `1` and also accept `full` (which would propagate as `99`) or integer. That should scale to custom plugins as well as it should be easy to use.